### PR TITLE
create workspace using service account prisma1

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix astronomer/issues#1851

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
practically LGTM:
```
# Write your query or mutation here
mutation CreateWorkspace {
  createWorkspace(
    label: "newasdasd1234444asdasd"
    description: "<workspace description>"
  ) {
    id
    label
  }
}

{
  "data": {
    "createWorkspace": {
      "id": "ckfyihx8p00600877zavk3szh",
      "label": "newasdasd1234444asdasd"
    }
  }
}
```

**Review by kushalmalani — APPROVED** [review]

File: src/resolvers/mutation/create-workspace/index.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ export default async function createWorkspace(parent, args, ctx, info) {
     .add(trialDuration, "d")
     .format();
 
+  const connectKey = get(ctx, "user.apiKey") ? "serviceAccount" : "user";
```
Comment (schnie) [inline src/resolvers/mutation/create-workspace/index.js:22]: @andriisoldatenko are we setting `user.apiKey` already?

**Review by schnie — APPROVED** [review]

**Review by adam2k — APPROVED** [review]
Looks good.  I guess we can't do anything about the snyk issues because this is Prisma 1, right?

File: src/resolvers/mutation/create-workspace/index.js (hunk 1 above)
Comment (andriisoldatenko) [reply src/resolvers/mutation/create-workspace/index.js:22]: no we don't added in https://github.com/astronomer/houston-api/pull/491/files#diff-2207f16fd110b62e422334ce89c57a9dR4

**Pull request merged**